### PR TITLE
Add notes section for bookings

### LINKED ISSUE #118: As a user, I would like to add notes to my bookings
so that I can remember important information about them.

### PR DESCRIPTION
**Pull request by jerremyng**
Closes #118

Simple implementation of notes to allow user to record important information.

### CODE REVIEW

**Comment by cyqjoseph** [comment]
Looks good, maybe add more test cases for Notes!

**Comment by jerremyng** [comment]
> Looks good, maybe add more test cases for Notes!

Added some test cases for notes!

File: build.gradle [hunk 1]
```diff
@@ -74,3 +74,7 @@ shadowJar {
 }
 
 defaultTasks 'clean', 'test'
+
```
Comment (cyqjoseph) [inline build.gradle:77]: Great!

File: src/test/java/seedu/address/logic/parser/BookCommandParserTest.java [hunk 2]
```diff
@@ -5,14 +5,29 @@
...
+        // No note is still valid
+        BookingBuilder bookingBuilder = new BookingBuilder();
+
+        Booking expectedBookingWithNote = bookingBuilder.build();
+        String userInputWithNote = bookingBuilder.getBookingString();
+
+        assertParseSuccess(parser,
+                userInputWithNote,
+                new BookCommand(expectedBookingWithNote));
+    }
```
Comment (cyqjoseph) [inline src/test/java/seedu/address/logic/parser/BookCommandParserTest.java:30]: Great!

**Review by cyqjoseph — COMMENTED** [review]
All good here!

**Pull request merged**